salt/docker_apps/files/pelican_backend.yaml

```diff
@@ -2,11 +2,6 @@ version: "2.2"
 
 x-shared: &shared
   image: "ghcr.io/open-contracting/pelican-backend:latest"
-  network_mode: "host"
```

Comment:
Pelican backend needs to communicate with PostgreSQL and RabbitMQ on the host, and it needs to make remote connections. Can it still do those without `network_mode: "host"`?

Reply:
Oh, I forgot about that. We could do the same as https://github.com/open-contracting/deploy/pull/316/commits/4cf8abb7b450f021eb2ab746e9e002ac9180c111 to still be able to communicate with the host's Postgres and RabbitMQ services, or we can leave this the network_mose as "host".

Reply:
We should have the same approach to `network_mode` across all files. We can either change it now to use the approach in 4cf8abb, or open a separate issue.

Reply:
I will leave the network_mode as it is here, and then we can follow up on the existing issue https://github.com/open-contracting/deploy/issues/307